molvis/vasp/python/vasputil.py

```diff
@@ -105,7 +105,7 @@ def parseFile(file, flipSign=False, centerData=True):
     volume.axes[2].name = "z"
     volume.axes[2].unit = ivw.data.Unit("Angstrom" )
     volume.dataMap.valueAxis.name = "Charge Density"
-    volume.dataMap.valueAxis.unit = ivw.data.Unit("eV")
+    volume.dataMap.valueAxis.unit = ivw.data.Unit("e/A^3")
```

Comment:
I don't think this is right, at least not for chgcar. 

> The structure block uses the same format as the [POSCAR](https://www.vasp.at/wiki/index.php/POSCAR) file. The total charge density is represented on the fine FFT-grid ([NGXF](https://www.vasp.at/wiki/index.php/NGXF),[NGYF](https://www.vasp.at/wiki/index.php/NGYF),[NGZF](https://www.vasp.at/wiki/index.php/NGZF)) and is multiplied by the cell volume, i.e., n(r)∗Vcell{\displaystyle n(r)*V_{\rm {cell}}} (internally VASP stores the density times the volume). Thus, remember that the charge density should be divided by the cell volume for visualization. As the density is written out using the following command in Fortran

https://www.vasp.at/wiki/index.php/CHGCAR